code/API_definitions/device-status.yaml

```diff
@@ -205,6 +398,205 @@ components:
         message:
           type: string
           description: Detailed error description
+    DeviceStatusEventType:
+      description: Type of the event
+      type: string
+      enum:
+        - ROAMING_STATUS
+        - ROAMING_ON
+        - ROAMING_OFF
+        - ROAMING_CHANGE_COUNTRY
+    DeviceStatusEventTypeList:
+      description: List of Device Status Event Type
+      type: array
+      items:
+        $ref: '#/components/schemas/DeviceStatusEventType'
+      example: [ ROAMING_STATUS]
+    CreateSubscription:
+      description: The request for creating a Device Status Subscription
+      type: object
+      properties:
+        subscriptionDetail:
+          $ref: '#/components/schemas/SubscriptionDetail'
+        subscriptionExpireTime:
+          type: string
+          format: date-time
+          example: 2023-01-17T13:18:23.682Z
+          description: The subscription expiration time in date-time format.
+        webhook:
+          $ref: '#/components/schemas/Webhook'
+    Webhook:
+      description: Webhook information for event channel
```

Comment:
According to the guidlines it's mandatory.

Reply:
Fixed